tests/unittests/sources/test_ibmcloud.py

```diff
@@ -272,6 +272,8 @@ def test_template_live(self, m_platform, m_sysuuid):
         )
 
         ret = ibm.read_md()
+        if ret is None:  # this is needed for mypy - ensures ret is not None
+            self.fail("read_md returned None unexpectedly")
```

Comment:
@TheRealFalcon is there a better way I should be doing this? I think this is pretty okay but just curious. thanks!

Reply:
I'm tempted to disable mypy for tests, but for mypy complaining this looks good!

Reply:
Well now that you mention it, mypy is supposed to be disabled for tests. All I did in this PR was remove the line disabling mypy checks for the ibm datasource file, not the unit test file for it.